### PR TITLE
CPG-968: Added hocon template

### PR DESCRIPTION
**Pull request by gAmUssA**
### Description 

This PR adds two templates that support HOCON configuration format.
Used in Scala's Play Framework and Kotlin's Ktor framework.

### Author Validation

_Describe the validation already done, or needs to be done, by the PR submitter._

### Reviewer Tasks

_Describe the tasks/validation that the PR submitter is requesting to be done by the reviewer._

### CODE REVIEW

**Review by ybyzek — CHANGES_REQUESTED** [review]
All the other files in this directory have a `.config` extension, can you please rename these 2 files to have the same extension, for consistency?

File: clients/docs/includes/configs/cloud/hocon-sr.conf [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  properties {
+    security.protocol = SASL_SSL
+    sasl.jaas.config = "org.apache.kafka.common.security.plain.PlainLoginModule   required username='{{ CLUSTER_API_KEY }}'   password='{{ CLUSTER_API_SECRET }}';"
+    sasl.mechanism = PLAIN
+    # Required for correctness in Apache Kafka clients prior to 2.6
+    client.dns.lookup = use_all_dns_ips
+    # Best practice for Kafka producer to prevent data loss
+    acks = all
+
+    //# Required connection configs for Confluent Cloud Schema Registry
```
Comment (laura-brouckman) [inline clients/docs/includes/configs/cloud/hocon-sr.conf]: should this just be # (remove the //)?

**Review by laura-brouckman — APPROVED** [review]
lgtm after Yeva's comment

File: clients/docs/includes/configs/cloud/hocon-sr.conf (hunk 1 above)
Comment (gAmUssA) [reply clients/docs/includes/configs/cloud/hocon-sr.conf]: fixed @laura-brouckman

**Comment by gAmUssA** [comment]
> All the other files in this directory have a `.config` extension, can you please rename these 2 files to have the same extension, for consistency?

in general, I don't have a strong opinion on this but in ktor they use `application.conf` and my module follows same pattern for configuration - `kafka.conf`. Let me know if it's a showstopper @ybyzek

**Comment by ybyzek** [comment]
@gAmUssA maybe not a showstopper, but a quick clarification question: how/where will these be used, where will they be surfaced?  Answer will help determine if consistency is super important or maybe not.

**Comment by gAmUssA** [comment]
> @gAmUssA maybe not a showstopper, but a quick clarification question: how/where will these be used, where will they be surfaced?

 
@ybyzek 
Firstly, copy from cloud UI, paste into IDE.
I want to add this to `ccloud-stack` as well (but later)

So if it's internal only and we can put whatever description in UI I will just go ahead and rename it

**Comment by ybyzek** [comment]
@gAmUssA If you want this picked up by CCloud UI, then engineering needs to be a part of this as well (cc: @laura-brouckman) and for that reason, I think consistency is important and my vote would be to rename the files.  FWIW I often see `conf`, `config`, and `cfg` interchanged, so hopefully it's ok.

**Comment by gAmUssA** [comment]
@ybyzek a standard behaviour of the library described here https://github.com/lightbend/config#standard-behavior
They don't use `cfg` convention.

**Comment by gAmUssA** [comment]
@laura-brouckman @ybyzek do we still have any concerns?

**Comment by ybyzek** [comment]
> @gAmUssA If you want this picked up by CCloud UI, then engineering needs to be a part of this as well (cc: @laura-brouckman) and for that reason, I think consistency is important and my vote would be to rename the files. 

Yes, please rename to `.config` for consistency

**Comment by gAmUssA** [comment]
> Yes, please rename to `.config` for consistency

@ybyzek ✅ done

**Review by ybyzek — APPROVED** [review]
LGTM! 👍

**Pull request merged**